CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 # prettier-plugin-svelte changelog
 
+## 3.0.0 (Unreleased
+
+-   (breaking) requires `prettier` version 3
+-   (breaking) requires node version 14 or higher
```

Comment:
should we use `:` for consistency with the other repos? should probably capitalize Node
```suggestion
-   breaking: requires Node version 14 or higher
```